tests/testthat/test-shrink.R

```
@@ -1,5 +1,24 @@
-test_that("shrink works", {
-  old_img <- array(1:24, dim = c(3, 4, 2))
 
-  expect_identical(sharpen(old_img), old_img)
+image <- array(1:24, dim = c(50, 50, 3))
```

The dimension is off; this array should throw an error.

---

I misunderstood how `array()` works; should be fine